snackbars/src/main/java/com/decathlon/vitamin/compose/snackbars/VitaminSnackbars.kt

```diff
@@ -0,0 +1,70 @@
+package com.decathlon.vitamin.compose.snackbars
+
+import androidx.compose.material.Snackbar
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminSnackbars {
+    @Composable
+    fun Primary(
+        text: String,
+        modifier: Modifier = Modifier,
+        action: @Composable (VitaminSnackbarsActions.() -> Unit)? = null,
+        actionOnNewLine: Boolean = false
+    ) = Snackbar(
+        modifier = modifier,
+        action = { action?.let { VitaminSnackbarsActions.it() } },
+        actionOnNewLine = actionOnNewLine,
+        backgroundColor = VitaminTheme.colors.vtmnBackgroundPrimaryReversed,
+        contentColor = VitaminTheme.colors.vtmnContentPrimaryReversed,
```

Comment:
Create a `SnackbarColors` class and `VitaminSnackbarColors` object to handle colors like other components. Check `buttons` module or my current PR which apply our compose convention on all modules.